web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -23,10 +26,20 @@
     canCopyImagesToClipboard = module.canCopyImagesToClipboard;
   });
 
-  const loadAssetData = async () => {
+  /**
+   * Loads the asset data from the API.
+   * @param forceWebImage - If true, the web version of the image will be loaded even if
+   * it's already in a format commonly supported by web browsers.
+   */
+  const loadAssetData = async (forceWebImage?: boolean) => {
     try {
       const { data } = await api.assetApi.serveFile(
-        { id: asset.id, isThumb: false, isWeb: true, key: api.getKey() },
+        {
+          id: asset.id,
+          isThumb: false,
+          isWeb: forceWebImage || !isImageCommonlySupportedByWeb(asset),
```

Comment:
`isWeb` is apparently just supposed to mean `Is request made from web` (rather than mobile), but both this and `isThumb` are legacy flags that don't seem all that necessary. Do we want to repurpose this flag to mean "serve web-compatible asset" like this? @jrasm91

Reply:
Idk. The whole file serving thing needs to be cleaned up a bit. I'm not even 100% sure what these flags even do tbh.